src/macaron/code_analyzer/call_graph.py

```diff
@@ -41,17 +43,18 @@ def has_callee(self) -> bool:
 
 
 class CallGraph(Generic[Node]):
-    """This is the generic class for creating a call graph.
-
-    Parameters
-    ----------
-    root : Node
-        The root call graph node.
-    repo_path : str
-        The path to the repo.
-    """
+    """This is the generic class for creating a call graph."""
 
     def __init__(self, root: Node, repo_path: str) -> None:
```

Comment:
Putting the documentation inside `__init__` might not be rendered in the generated Sphinx documentation at the moment. I think we could tell Sphinx to display it according to the conversation here - https://stackoverflow.com/a/5599712

Reply:
> Putting the documentation inside `__init__` might not be rendered in the generated Sphinx documentation at the moment. I think we could tell Sphinx to display it according to the conversation here - https://stackoverflow.com/a/5599712

Thanks, I'll add the change to be able to render the docstrings in `__init__` methods.